Automattic/wp-calypso
# Composite checkout: add new credit card logos

#### Changes proposed in this Pull Request

This PR adds additional logos for a handful of Stripe supported credit cards. 

![image](https://user-images.githubusercontent.com/6981253/68957911-072cf900-0799-11ea-98b2-0dfd5e8cd4b8.png)

![image](https://user-images.githubusercontent.com/6981253/68957935-16ac4200-0799-11ea-81d6-390479efb10d.png)


#### Testing instructions
* Load up the checkout demo by following these instructions: https://github.com/Automattic/wp-calypso/pull/37013
* Open the second Credit Card payment method and add the following combinations to see the logos:
 * 34, 37 > AMEX
 * 35 > JCB
 * 36, 38, 39 > Diners
 * 4 > Visa
 * 51 - 55 > Mastercard
 * 60, 64, 65 > Discover
 * 62 > Unionpay
